nri-config-generator/integration-tests/generator_test.go

```diff
@@ -162,13 +164,14 @@ func TestGeneratorConfigWithInterval(t *testing.T) {
 	assert.JSONEq(t, expectedResponse, string(stdout))
 }
 
-/**
+/*
+*
 The verbose mode of the Agent gets propagated to exporter and prometheus
 */
 func TestGeneratorVerboseMode(t *testing.T) {
 	envVars := getConfigGeneratorEnvVars("config.yml")
 	envVars = append(envVars, "VERBOSE=1")
-	stdout, _ := callGeneratorConfig(testIntegration, defaultArgs, envVars)
+	stdout, _ := callGeneratorConfig(defaultArgs, envVars)
```

Comment:
Nit: maybe we could take the chance to check the error instead of ignoring it. `require.NoError` maybe?